### ISSUE TITLE
Gets stuck after running command

### ISSUE THREAD

**Issue by talhah58** [m1stadev/deleo#7]
After i run the command it just gets stuck and seemingly does nothing. 

![image](https://github.com/m1stadev/deleo/assets/113779460/af470292-30bd-4ab9-a26c-6580af810a03)

**Comment by m1stadev** [comment]
do you have usbmuxd installed (and running)?

**Comment by talhah58** [comment]
yeah i do have usbmuxd running and I'll try again with the correct command again today

**Comment by talhah58** [comment]
yeah i tried restarting usbmuxd and used the correct command but it still wont work

**Comment by talhah58** [comment]
Where would i find the log file? 

The command is 

deleo --ecid 133790091438202 -u -v -t blob.shsh2 16.6.1.ipsw latest.ipsw

btw it's an iPhone X upgrading from 15.4.1 to 16.6.1

**Comment by m1stadev** [comment]
> Where would i find the log file?

it's just the output of the command (if there is any)

can you also run `pymobiledevice3 usbmux list` with the device connected & booted in normal mode and see if the device shows up?

**Comment by talhah58** [comment]
yeah there is no output after running the command (it just gets stuck) except when i press ctrl+c is says "Aborted!"
This is also happening when i run  pymobiledevice3 usbmux list

**Comment by talhah58** [comment]
actually wait nvm it does come up with something after waiting a few seconds

**Comment by m1stadev** [comment]
> can you also run `pymobiledevice3 usbmux list` with the device connected & booted in normal mode **and see if the device shows up?**

you haven't answered the question

**Comment by m1stadev** [comment]
can you install the latest version from github (`pip install -U git+https://github.com/m1stadev/deleo.git`), and try restoring again?

**Comment by kixrd** [comment]
it now gives me this errror


![image](https://github.com/m1stadev/deleo/assets/113779460/fc2d5b45-7726-41fe-be39-dc1c8b4aee9a)

**Comment by kixrd** [comment]
okay i reinstalled deleo and its getting stuck the same way it did before

**Comment by kixrd** [comment]
except this shows up when i press ctrl+c now 

![image](https://github.com/m1stadev/deleo/assets/113779460/9e12cbc8-cd8c-4566-a45a-ecdde76e4d5b)

**Comment by kixrd** [comment]
Nvm i reinstalled deleo and its the same as before
